crates/ark/src/modules/positron/viewer.R

```diff
@@ -6,11 +6,28 @@
 #
 
 options("viewer" = function(url, height = NULL, ...) {
+    # Normalize paths for comparison. This is necessary because on e.g. macOS,
+    # the `tempdir()` may contain `//` or other non-standard path separators.
+    normalizedPath <- normalizePath(url, mustWork = FALSE)
```

Comment:
```suggestion
    # Normalize paths for comparison. This is necessary because on e.g. macOS,
    # the `tempdir()` may contain `//` or other non-standard path separators.
    # See https://github.com/posit-dev/positron/issues/3018.
    normalizedPath <- normalizePath(url, mustWork = FALSE)
```

This fixes https://github.com/posit-dev/positron/issues/3018 right? You could include that in your list of issues that your Positron side PR addresses.

And https://github.com/posit-dev/positron/issues/3269 since it was probably the same underlying issue

Reply:
It does fix 3018! I'll resolve that one with the PR.

it doesn't fix 3269. :-(